sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -33,6 +33,11 @@
 #include "debug.h"
 
 /**
+ * @brief Helper macro for LL_SEARCH to compare TCBs
+ */
+#define TCB_EQUAL(a,b)      ((a) == (b))
+
+/**
```

Comment:
Shouldn't this be a function (maybe inline)? 

According to 
http://www.cs.bu.edu/~jappavoo/Resources/psml/apps/hashy/uthash/doc/utlist.html

LL_SEARCH expects a pointer to a function not a macro